fabric-content-registries-v0/src/main/java/net/fabricmc/fabric/mixin/content/registry/SculkSensorBlockMixin.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registry;
+
+import it.unimi.dsi.fastutil.objects.Object2IntMap;
+import org.spongepowered.asm.mixin.Dynamic;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Redirect;
+
+import net.minecraft.block.SculkSensorBlock;
+
+@Mixin(SculkSensorBlock.class)
+public class SculkSensorBlockMixin {
+	@Dynamic("Call to Object2IntMaps.unmodifiable in initialization of FREQUENCIES")
```

Comment:
Why dynamic?

Reply:
It's a decorator annotation provided by mixin specifically for providing context for mixins that aren't immediately obvious where they modify. Similar usage is found in this module's `OxidizableMixin` and `HoneycombMixin` when they inject into lambda bodies. I suppose `<clinit>` isn't altogether hard to imagine, but it's certainly less transparent than a mixin to a visible method, and so I felt it somewhat warranted.

Reply:
It should only be used for lambda bodies imo, `<clinit>` is quite well known and probably doesn't require the annotation.

Reply:
This isn't resolved yet since `@Dynamic` is still there? I don't think it makes sense to have it.